core/src/main/java/org/apache/accumulo/core/conf/ConfigSanityCheck.java

```diff
@@ -51,6 +54,8 @@ public static void validate(Iterable<Entry<String,String>> entries) {
     boolean usingVolumes = false;
     String cipherSuite = null;
     String keyAlgorithm = null;
+    String secretKeyEncryptionStrategy = null;
+    String cryptoModule = null;
```

Comment:
Should these values be set to the new constants above?

Reply:
Yeah. In fact, that will clean up the unit test a little too.